charts/newrelic-logging/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: A Helm chart to deploy New Relic Kubernetes Logging as a DaemonSet, supporting both Linux and Windows nodes and containers
 name: newrelic-logging
-version: 1.21.5
-appVersion: 1.19.2
+version: 1.21.6
```

Comment:
I'd be tempted to bump major here, at least minor, definetely not patch, wdyt?

Reply:
updated to minor version